src/main/java/net/javadiscord/javabot/data/h2db/message_cache/MessageCacheListener.java

```diff
@@ -67,8 +67,9 @@ public void onMessageDelete(@NotNull MessageDeleteEvent event) {
 	 * @return true if any of the criteria above apply
 	 */
 	private boolean ignoreMessageCache(Message message) {
+		if (!message.isFromGuild()) return true;
 		MessageCacheConfig config = Bot.config.get(message.getGuild()).getMessageCache();
-		return !message.isFromGuild() || message.getAuthor().isBot() || message.getAuthor().isSystem() ||
+		return message.getAuthor().isBot() || message.getAuthor().isSystem() ||
```

Comment:
This was done because `message#getGuild` throws an exception if the message was not sent in a guild. This would happen when direct messages are sent/received